symfony/plugins/orangehrmMarketPlacePlugin/lib/manager/APIManagerService.php

```diff
@@ -314,11 +314,6 @@ public function handShakeWithMarketPlace()
     {
         $instanceIdentifier = $this->getConfigService()->getInstanceIdentifier();
         $instanceIdentifierChecksum = $this->getConfigService()->getInstanceIdentifierChecksum();
-        if (empty($instanceIdentifier)){
```

Comment:
This part may need if the instance identifier is empty and calling to MP .. 
This is the way that we fix .EXE installation in the last release